### DISCUSSION TITLE
中枢上移有哪些必要条件分析

### DISCUSSION THREAD

**Discussion by zengbin93** [waditu/czsc#36] [General]
以日线为例，讨论中枢上移过程必然出现的一些情况。

**Comment by skanda-huayan** [comment]
出中枢的一段，次级别出现3买，大概率走强；
然后在本级别出现3买。回调段，不跌破前中枢GG最强，不跌破前中枢ZG次之。

**Reply by zengbin93** [reply]
三买是中枢上移的必要条件之一。

**Comment by zengbin93** [comment]
## 中枢上移过程的必要条件1：MACD指标中的 DIFF 和 DEA 都不可能跌破零轴

**Comment by zengbin93** [comment]
## 中枢上移过程的必要条件2：三买后的每一段次级别走势都必须创新高且无背驰

**Comment by zengbin93** [comment]
A股股票出现日线中枢上移，通常都必须有基本面或消息面的共振。
